### DISCUSSION TITLE
Helm vs Kustomize RBAC

### DISCUSSION THREAD

**Discussion by cyrus-mc** [bank-vaults/vault-operator#153] [Q&A] [answered]
Why are the RBAC rules defined here - https://github.com/bank-vaults/vault-operator/tree/main/deploy/rbac so different than what the Helm Chart generates here - https://github.com/bank-vaults/vault-operator/blob/main/deploy/charts/vault-operator/templates/role.yaml

**Comment by akijakya** [comment]
Summon @ramizpolic 🙂

**Reply by ramizpolic** [reply, marked as answer]
https://github.com/bank-vaults/vault-operator/tree/main/deploy/rbac define RBAC generated by kubebuilder for kustomize support. Note that these policies are incomplete since we just recently started working on this. We will slowly be updating them to reflect the same policies as defined in https://github.com/bank-vaults/vault-operator/blob/main/deploy/charts/vault-operator/templates/role.yaml. 

Currently, only helm installation is supported.
